tools/pipelines/templates/build-npm-package.yml

```diff
@@ -277,6 +281,33 @@ stages:
               tagName: ${{ parameters.tagName }}
               interdependencyRange: ${{ parameters.interdependencyRange }}
 
+        # Generate rollups
+        - ${{ if or(eq(parameters.releaseBuildOverride, 'prerelease - alpha'), eq(parameters.releaseBuildOverride, 'prerelease - beta')) }}:
```

Comment:
I don't think you want to use the override value. Instead, the override value should control the release type, which is a different pipeline-wide variable:

https://github.com/microsoft/FluidFramework/blob/7dbf965ade9b1e947b122ca3eaeacf16f604a37d/tools/pipelines/templates/include-vars.yml#L93-L103

I think you're ultimately trying to add new values to that variable.

Reply:
Actually, it's might be better to make a separate variable.